test/board_test.rb

```diff
@@ -76,4 +76,44 @@ def test_it_can_not_place_ship_on_top_of_ship
     refute board.valid_placement?(submarine, ["D1", "D2"])
     assert board.valid_placement?(submarine, ["A1", "A2"])
   end
+
+  def test_it_can_render_correctly
```

Comment:
These are some awesome tests! Maybe this is a great opportunity to look into using another setup method that runs these setup lines(81, 82,83)